examples/horovod/tensorflow-mnist-elastic.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubeflow.org/v1
+kind: MPIJob
+metadata:
+  name: tensorflow-mnist
+spec:
+  slotsPerWorker: 1
+  cleanPodPolicy: Running
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        spec:
+          containers:
+          - image: docker.io/kubeflow/mpi-horovod-mnist
```

Comment:
Is there a Dockerfile for this as a reference? Users might want to customize it.

Reply:
In particular, they'd be interested in what versions of TF and Horovod they should be using in this image.

Reply:
I'm trying to build the docker image with Horovod version jumped to 0.20.0. It failed awkwardly. Given horovod has its own official image, any merits to build one by kubeflow itself?

Reply:
I think it's fine to add a link to their official Dockerfiles/build instructions. We've seen people asking for a Dockerfile before regarding one of the examples in the repo.

Reply:
Another way is we simply change the dockerfile.cpu, removing the build phase. In that way, the dockerfile.cpu will only add example scripts to `horovod/horovod:0.20.0-tf2.3.0-torch1.6.0-mxnet1.5.0-py3.7-cpu`.

Reply:
I think we can do it in another PR, WDYT

Reply:
Yes that's fine.